### PR TITLE
[更新完毕] dev_install

### PR DESCRIPTION
**Pull request by ArkhamWJZ**

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 1]
```diff
@@ -7,302 +7,348 @@ msgid ""
...
+"Ubuntu 16.04+ and OS X 10.14+. For precise build instructions on Windows, please "
+"check out `appveyor.yml <https://github.com/taichi-dev/taichi/blob/master/"
+"appveyor.yml>`_, which does basically the same thing as the following "
+"instructions. We use MSBUILD.exe to build the generated project. Please note "
+"that Windows could have multiple instances of MSBUILD.exe shipped with different "
+"products. Please make sure you add the path for MSBUILD.exe within your MSVS "
+"directory and make it a higher priority (for instance than the one shipped with ."
+"NET)."
+msgstr ""
+"请注意，本章节是为 Taichi 编程语言的编译器开发人员所准备。终端用户应该使用 pip 进"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
"请注意，本章节是为 Taichi 语言编译器的开发人员所准备的。终端用户应该使用 pip 进"
```

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 2]
```diff
@@ -7,302 +7,348 @@ msgid ""
...
-"sources probably won't work. Here we provide LLVM binaries customized for"
-" Taichi, which may or may not work depending on your system environment:"
+"Make sure you have LLVM 10.0.0. Note that Taichi uses a **customized LLVM** so "
+"the pre-built binaries from the LLVM official website or other sources probably "
+"won't work. Here we provide LLVM binaries customized for Taichi, which may or "
+"may not work depending on your system environment:"
 msgstr ""
+"确保你已经安装了 LLVM 10.0.0。注意，Taichi 使用的是 **定制版 LLVM**，所以从 LLVM "
+"官网或其他来源下载的预构建可执行文件可能不会工作。这里我们提供了 Taichi 的定制版 "
+"LLVM 可执行文件，它是否能正常工作取决于你的系统环境:"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
"确保你已经安装了 LLVM 10.0.0。注意，Taichi 使用的是 **定制版 LLVM**，所以从 LLVM "
"官网或其他来源下载的预构建可执行文件可能不起作用。这里我们提供了 Taichi 的定制版 "
"LLVM 可执行文件，它是否能正常运行取决于你的系统环境:"
```

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 3]
```diff
@@ -7,302 +7,348 @@ msgid ""
...
+"了。"
 
 #: ../../dev_install.rst:124
 msgid ""
-"Add or append ``PATH`` with path to LLVM binary directory installed in "
-"previous section."
-msgstr ""
+"Add or append ``PATH`` with path to LLVM binary directory installed in previous "
+"section."
+msgstr "添加或追加之前小节安装的 LLVM 可执行文件路径到 ``PATH``。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
msgstr "添加或附加之前章节安装的 LLVM 可执行文件路径到 ``PATH``。"
```

**Review by isdanni — COMMENTED** [review]
🚀 🚀 🚀

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 4]
```diff
@@ -7,302 +7,348 @@ msgid ""
...
+"check out `appveyor.yml <https://github.com/taichi-dev/taichi/blob/master/"
+"appveyor.yml>`_, which does basically the same thing as the following "
+"instructions. We use MSBUILD.exe to build the generated project. Please note "
+"that Windows could have multiple instances of MSBUILD.exe shipped with different "
+"products. Please make sure you add the path for MSBUILD.exe within your MSVS "
+"directory and make it a higher priority (for instance than the one shipped with ."
+"NET)."
+msgstr ""
+"请注意，本章节是为 Taichi 语言编译器的开发人员所准备的。终端用户应该使用 pip 进行"
+"安装，而不是从源代码开始构建。为支持在 NVIDIA GPU 上进行工程构建，CUDA 10.0+是必"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
"安装，而不是从源代码开始构建。要在构建加入对 NVIDIA GPU 的支持，CUDA 10.0+是必"
```
Btw, @yuanming-hu I thought building with CUDA support doesn't need CUDA thanks to our `DynamicLoader`?

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 5]
```diff
@@ -7,302 +7,348 @@ msgid ""
...
-"On Windows, if you use the pre-built LLVM for Taichi, please add "
-"``$LLVM_FOLDER/bin`` to ``PATH``. Later, when you build Taichi using "
-"``CMake``, set ``LLVM_DIR`` to ``$LLVM_FOLDER/lib/cmake/llvm``."
+"On Windows, if you use the pre-built LLVM for Taichi, please add ``$LLVM_FOLDER/"
+"bin`` to ``PATH``. Later, when you build Taichi using ``CMake``, set "
+"``LLVM_DIR`` to ``$LLVM_FOLDER/lib/cmake/llvm``."
 msgstr ""
+"在 Windows 下, 如果你使用的是 Taichi 版预构建的 LLVM, 请将 ``$LLVM_FOLDER/bin`` "
+"添加到 ``PATH``。随后，当你使用 ``CMake`` 构建 Taichi 时，将 ``LLVM_DIR`` 设置到 "
+"``$LLVM_FOLDER/lib/cmake/llvm``."
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
"在 Windows 下, 如果你使用的是 Taichi 版预构建的 LLVM, 请将 ``$LLVM_FOLDER/bin`` "
"添加到 ``PATH``。随后，当你使用 ``CMake`` 构建 Taichi 时，将 ``LLVM_DIR`` 设置为 "
"``$LLVM_FOLDER/lib/cmake/llvm``."
```

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 6]
```diff
@@ -315,79 +361,91 @@ msgid ""
...
+"minutes to run all tests."
+msgstr ""
+"执行 ``python3 -m taichi test`` 来运行所有的测试。运行所有测试用例可能会耗费长达5"
+"分钟的时间。"
 
 #: ../../dev_install.rst:149
-#, fuzzy
 msgid "Troubleshooting Developer Installation"
-msgstr "开发者安装指南"
+msgstr "安装中的故障排除"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
msgstr "开发者安装中的故障排除"
```

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 7]
```diff
@@ -315,79 +361,91 @@ msgid ""
...
 #: ../../dev_install.rst:151
 msgid ""
-"If ``make`` fails to compile and reports ``fatal error: 'spdlog/XXX.h' "
-"file not found``, please try runing ``git submodule update --init "
-"--recursive --depth=1``."
+"If ``make`` fails to compile and reports ``fatal error: 'spdlog/XXX.h' file not "
+"found``, please try runing ``git submodule update --init --recursive --depth=1``."
 msgstr ""
+"如果编译时 ``make`` 失败并且报告 ``fatal error: ‘spdlog/XXX.h’ file not found``, "
+"请尝试执行 ``git submodule update —init —recursive —depth=1``."
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
"请尝试执行 ``git submodule update --init --recursive --depth=1``."
```

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 8]
```diff
@@ -315,79 +361,91 @@ msgid ""
...
 msgstr ""
 
 #: ../../dev_install.rst:161
 msgid ""
-"Please try adding ``TAICHI_REPO_DIR`` to environment variables, see "
-":ref:`dev_env_settings`."
+"Please try adding ``TAICHI_REPO_DIR`` to environment variables, see :ref:"
+"`dev_env_settings`."
 msgstr ""
+"请尝试将 ``TAICHI_REPO_DIR`` 添加到环境变量中, 详见 :ref:`dev_env_settings`."
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
"请尝试将 ``TAICHI_REPO_DIR`` 添加到环境变量中, 详见 :ref:`dev_env_settings` 。"
```

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 9]
```diff
@@ -397,82 +455,101 @@ msgid ""
 "# should be /usr/local/bin/llvm-as or /opt/XXX/bin/llvm-as, which is our "
 "configured installation"
 msgstr ""
+"llvm-as —version\n"
+"# 版本号应该 >= 8\n"
+"which llvm-as\n"
+"# 应该输出 /usr/local/bin/llvm-as 或 /opt/XXX/bin/llvm-as, 这是我们的配置文件所在"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
"# 应该输出 /usr/local/bin/llvm-as 或 /opt/XXX/bin/llvm-as, 这是我们的定制版 LLVM 所在"
```

**Review by archibate — CHANGES_REQUESTED** [review]

**Pull request merged**